### ISSUE TITLE
Possible to enter state where Account.__repr__() results in Type error in attempting to serialize Datetime object

### ISSUE THREAD

**Issue by dalanmiller** [stripe/stripe-python#374]
In attempting to make mutations to an Account object, I can set the following:

```
account = stripe.Account.retrieve("acc_123") 
account['tos_acceptance'] = {
     'date': datetime.datetime(year = 2017, month = 11, day=16),
     'ip': '192.168.1.1' #For example's sake
    }
```

Now, if I run `account.__repr__()` (or just `account` in some sort of REPL, I get the following: 

```
TypeError: Object of type 'datetime' is not JSON serializable
```

Given that in the `Account.save()` method, Python `datetime` objects automatically get translated into Stripe API comprehensible timestamps, it would probably make sense to do this within the `__repr__` as well for inspection in iPython, ipynb, IDEs or elsewhere I'm not thinking of.

**Comment by ob-stripe** [comment]
Makes sense. Thanks for the report @dalanmiller :)

**Comment by ob-stripe** [comment]
Fixed in v1.75.1.
